odom_frame_id: /odom
map_frame_id: /map
joint_states_topic: /robospect/joint_states
odom_topic: /robospect_platform_controller/odom
robot_state_topic: /robospect_platform_controller/state
# Link to use for the transformation
tip_link: tip_link
# Joints ordered from min to max
crane_joints: 
 - crane_first_joint
 - crane_second_joint
 - crane_third_joint
 - crane_fourth_joint
 - crane_fifth_joint
 - crane_sixth_joint
 - crane_tip_joint
